### ISSUE TITLE
Creating Mentors Introducing page

### ISSUE THREAD

**Issue by Bhawna11agg** [HakinCodes/CodeAsylums-App#40]
**Description of the enhancement**
I want to create the mentor introduction page using flutter

**Screenshots**
![image](https://user-images.githubusercontent.com/47032273/90670935-27389e00-e27e-11ea-87f6-8791e233b393.png)

**Comment by Bhawna11agg** [comment]
Can I work on this?

**Comment by champ96k** [comment]
@Bhawna11agg  yes you can work there, but please take care of navigation bar because in original website it's look different

**Comment by ashishsahu1** [comment]
Is this issue assigned or should I work on this?

**Comment by Bhawna11agg** [comment]
I am working on this

**Comment by helios1101** [comment]
Hi @ashishsahu1 
You can too work on this, pertaining to different platform i.e, react-native.
As @Bhawna11agg is working in flutter.
